content/general/concepts/algorithm/BigO.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: 'Big-O Notation'
```

Comment:
In order to properly nest under "algorithm" this file needs to be relocated to the following path:
content/general/concepts/algorithm/terms/BigO/BigO.md. This will allow Docs to properly put a link in the "algorithm" entry.

Reply:
@SSwiniarski I actually fixed this myself 😄 (noted [here](https://github.com/Codecademy/docs/pull/823#issuecomment-1166311367))